common/src/IO/NodeSerializer.cpp

```diff
@@ -194,7 +194,19 @@ namespace TrenchBroom {
         }
 
         String NodeSerializer::escapeEntityAttribute(const String& str) const {
-            return StringUtils::escape(str, "\"", '\\');
+            // just escape bare " characters
+            StringStream ss;
```

Comment:
Please move this into a separate function in StringUtils, e.g. `escapeIfNecessary`